extensions/amp-story/1.0/amp-story-page-attachment.css

```diff
@@ -37,6 +37,7 @@
   font-size: 15px !important;
   justify-content: space-between !important;
   line-height: 48px !important;
+  text-decoration: none !important;
```

Comment:
this only applies to the title of the page attachment, right?

Reply:
Yeah it's specific to the `remote` attachment. :))